packaging/hoodie-hive-bundle/pom.xml

```diff
@@ -163,10 +163,6 @@
             </goals>
             <configuration>
               <relocations>
```

Comment:
Can you manually verify if the resulting bundle do not have classes starting with com.beust.*

Reply:
@bvaradar com.beust.* is needed in resulting bundle so that AbstractConfig class can access config variables with com.beust.jcommander.Parameter annotations.

Reply:
Once you move AbstractConfig to hoodie utilities, this can be cleaned.